### DISCUSSION TITLE
Support of Radiant ?

### DISCUSSION THREAD

**Discussion by LoulouCrypto** [coinfoundry/miningcore#1442] [General]
IS there a Radiant coin support in a next update ??

**Comment by oliverw** [comment]
1. This is NOT a bug but a question.
2. Sure, when is your PR coming?

**Reply by LoulouCrypto** [reply]
1 . Sorry 😅 made the mistake
2. I will try to intégrate it 🙏 first Time for me 😅 i will make somes tests

**Reply by LoulouCrypto** [reply]
And not sure how to do it lool

**Reply by LoulouCrypto** [reply]
Can you tell me how to add it ? 
It's a specific algorithm si IS there something more to do than add a other coin ?

**Comment by oliverw** [comment]
https://github.com/oliverw/miningcore/discussions/1466
